library/mageia

```diff
@@ -2,6 +2,10 @@ Maintainers: Juan Luis Baptiste <juancho@mageia.org> (@juanluisbaptiste)
 GitRepo: https://github.com/juanluisbaptiste/docker-brew-mageia.git
 
 Tags: 6, latest
-GitCommit: f60e1b20a5e31e72b4f49c878160dfa95f49a911
-GitFetch: refs/heads/dist
-Directory: 6
+Architectures: amd64, arm32v7
+amd64-GitCommit: e7ffd1ff77c5914b7f46a6cee2373acfb0bf60f5
+amd64-Directory: 6/x86_64
+amd64-GitFetch: refs/heads/dist
+arm32v7-GitCommit: e7ffd1ff77c5914b7f46a6cee2373acfb0bf60f5
+arm32v7-GitFetch: refs/heads/dist
+arm32v7-Directory: 6/armv7h1
```

Comment:
This is wrong, it should be `6/armv7hl`, right?

Reply:
Ahhhhhhhhh I was starting to pull my beard out on whats wrong, good catch !!